lib/rucio/common/logging.py

```diff
@@ -392,18 +393,47 @@ def rucio_log_formatter(process_name: Optional[str] = None) -> RucioFormatter:
 
 def setup_logging(application: Optional["Flask"] = None, process_name: Optional[str] = None) -> None:
     """
-    Configures the logging by setting the output stream to stdout and
-    configures log level and log format.
+    Configure logging sink/format/level.
+    Defaults to stdout (backwards compatible).
+    Dev/test/CI can override via:
```

Comment:
Is this meant in the sense that this overriding can only be done in dev/test/CI? because if it's just a matter of changing the config or env variables, i assume you could easily do this in prod too